web/src/navbar_alerts.js

```diff
@@ -173,6 +193,8 @@ export function initialize() {
         });
     }
 
+    maybe_show_empty_required_profile_fields_alert();
```

Comment:
Look more carefully at the construction of `initialize`; it's designed to never show more than one at once.

This new one can be last, after `check_profile_incomplete`

Reply:
But shouldn't the warning banner still be visible even if there is an info banner in the navbar?

Reply:
The navbar alerts system isn't designed to handle more than one alert at a time; various CSS things I expect to break. So they have an order of precedence; a normal user should never have more than 1 applicable at a time anyway.

Reply:
OK, I have changed the code accordingly.

Reply:
OK, I have changed the code accordingly.